webapp/webapp.v3.spec.yaml

```diff
@@ -1019,6 +1038,11 @@ components:
         oval_only:
           type: boolean
           enum: [true, false]
+        third_party:
```

Comment:
defined here but not used in vulnerabilities API

Reply:
It is. Vulnerabilities internally uses the updates api, and that api supports third party flag.